docs/developing.md

```diff
@@ -0,0 +1,37 @@
+Developing for FakeNet-NG
+-------------------------
+
+FireEye only:
+* Create branches directly under the `fireeye/` GitHub repository, not under a
+  private fork.
+
+All contributors:
+* Pull request comment should bear bulleted list for inclusion in change log.
```

Comment:
A Pull Request should be focused on one thing like a bug fix or adding one new feature. PR's containing say 10 different bug fixes or new features should be rejected. This allows for quicker code review and not holding up say 9 fixes because 1 of them was implemented incorrectly

Reply:
Fair point